src/main/scala/de/upb/cs/swt/delphi/instanceregistry/io/swagger/client/model/Event.scala

```diff
@@ -79,9 +80,21 @@ trait EventJsonSupport extends SprayJsonSupport with DefaultJsonProtocol with In
     }
 
   }
+  implicit val timestamp: JsonFormat[DateTime] = new JsonFormat[DateTime] {
```

Comment:
Please rename this val to ```timestampFormat``` to be consistent with the other formats. Also please add JavaDoc comments to this val.